scripts/benchmark.py

```diff
@@ -17,21 +17,27 @@
 
 
 class Benchmark:
-    def __init__(self, repeat, n_iter, verbose):
+    def __init__(self, repeat, n_iter, verbose, random_state):
```

Comment:
May be worth splitting out these small independent parts into separate preliminary PRs to make the main part easier to review.

Reply:
definitely.